src/gsi-ncdiag/conv.py

```diff
@@ -227,7 +227,7 @@ def toIODAobs(self, OutDir, clobber=True):
                     for lvar in LocVars:
                         if lvar == 'Station_ID':
                             tmp = self.df[lvar][idx]
-                            locKeys.append([''.join(tmp[a])
+                            locKeys.append([b''.join(tmp[a])
```

Comment:
List comprehension on L230 and 231 don't fit on 1 line? Makes it easy to read.

Reply:
You know what, that is an artifact of using the automated pep8 script to fix the pycodestyle complaints, with the indentation the line length is too long for the default option. When I combine these into one lib-pygsidiag.py file, I'll try to sort back out these wacky line wrappings.